sentinel/pkg/checker/event/app.go

```diff
@@ -16,8 +16,10 @@ import (
 const AlarmOffset = 3
 
 var FeedsToCheck = []FeedToCheck{}
+var PegPorToCheck = FeedToCheck{}
 var EventCheckInterval time.Duration
 var BUFFER = 1 * time.Second
+var POR_BUFFER = 1 * time.Minute
```

Comment:
how about adding unit to the variable name to ensure better readability and usability. That way we wont have to come back to the variable definition to know the unit: `BUFFER_SEC` and `POR_BUFFER_MIN`

Reply:
great suggestion, will do